Objects/object.c

```diff
@@ -2970,3 +2971,51 @@ _Py_SetRefcnt(PyObject *ob, Py_ssize_t refcnt)
 {
     Py_SET_REFCNT(ob, refcnt);
 }
+
+
+static PyObject* constants[] = {
+    &_Py_NoneStruct,                   // Py_NONE id
+    (PyObject*)(&_Py_FalseStruct),     // Py_FALSE id
+    (PyObject*)(&_Py_TrueStruct),      // Py_TRUE id
+    &_Py_EllipsisObject,               // Py_ELLIPSIS id
+    &_Py_NotImplementedStruct,         // Py_NOT_IMPLEMENTED id
+};
+
+PyObject*
+Py_GetConstant(unsigned int constant_id)
+{
+    if (constant_id <= Py_NOT_IMPLEMENTED) {
```

Comment:
Maybe initialize the table with the remaining five objects on first call? Or is it helpful to have the table in read-only memory? The need for a switch seems a shame.

Reply:
Fixed